src/css/components/Panel.scss

```diff
@@ -73,6 +73,49 @@ $Panel-header-text-color: $cu-info !default;
   border-bottom-right-radius: $border-radius-large;
 }
 
+.Panel-body:last-child .Panel-row:last-child {
+  border-bottom-left-radius: $border-radius-large;
+  border-bottom-right-radius: $border-radius-large;
+}
+
+
+// Table Specific Stylings
+.Panel-body[role="table"] .Panel-row[role="row"] {
+  border-bottom: 0.5px solid color-neutral(5);
+  border-top: 0.5px solid color-neutral(5);
+}
+
+.Panel-body[role="table"] .Panel-row[role="row"]:not(:first-child):hover {
+  border-bottom: 1px solid color-primary(4);
+  border-top: 1px solid color-primary(4);
+  background: color-primary(1) !important;
+}
+
+.Panel-body[role="table"] .Panel-row[role="row"]:first-child {
```

Comment:
Noticed the box-shadow styles are applying to the first child of the nested row section too. I think this should fix the problem by targeting only first-child rows that are direct descendants of the table element:
`.Panel-body[role="table"] > .Panel-row[role="row"]:first-child`

Basically just adding the direct descendant selector `>`